src/node/Main.hs

```diff
@@ -227,6 +237,8 @@ updateTriggerWorker = first pure $ worker mempty $ \_ -> do
 -- Wallet stuff
 ----------------------------------------------------------------------------
 
+#ifdef WITH_WALLET
+
```

Comment:
>  Or WITH_WEB implies WITH_WALLET?

Hmm, no, it doesn't, but let's temporary assume it does 😄 

The task was to fix build w/o web **and** wallet and I think I did it. Perhaps it won't compile for web xor wallet, but it's anyway better than it was before.